### ISSUE TITLE
Doomfist change suggestion 

### ISSUE THREAD

**Issue by NotDahmer** [MaxwellJung/ow1_emulator#251]
There's a different version of 6v6 remastered that has a very smooth Doomfist kit. BJZQ7 is the code

**Comment by MaxwellJung** [comment]
Thanks I’ll check it out in the evening.

Do you know how faithful it is to ow1 doomfist?

**Comment by NotDahmer** [comment]
I’ll be honest I didn’t play ow 1 but It should be the same as yours just that slam activates at any distance and uppercut is still the power block animation but in the air it’s only for a second or so you have enough time for a full charge punch. I kind of find it easier to aim the uppercut with this version.
P.S I haven’t played the recent updates so I’m not sure if it’s already been implemented

**Comment by MaxwellJung** [comment]
Oh no… is this the doom from 6v6 remastered? Last time I tried their doom, I was very disappointed with how unfaithful their doom was.

This gamemode doomfist has way more accurate movement/gravity (including how doom steps forward before uppercut and his gravity changing throughout the entire duration of the uppercut). There’s a lot more things that we do better with doomfist, but I won’t go into that…

You’re probably confused about slam not activating because you likely aren’t aware of the indicator slam vs arc slam mechanic in ow1 doomfist. Every ow1 doomfist player that played this gamemode has personally told me that this doom is way more accurate than 6v6 remastered doom. It’s just occasionally feels buggy/clunky because our doom is way more complex than theirs.

However, I will keep an open mind since things might have gotten updated.

**Comment by NotDahmer** [comment]
I do know about the different slams I just couldn’t tell you or give my opinion since I didn’t get a chance to feel it in ow 1. I do like both Dooms it’s just the 1 thing that’s different is I feel easier to string combos together. It’s just a preference thing tbh nothing serious

**Comment by MaxwellJung** [comment]
I just tried it and our version is still better.

Inaccuracies:
1) Slam only deals damage in increments of 25
2) Practically non existent indicator slam
3) Uppercut can be immediately cancelled (in OW1, you get locked into animation for ~0.2s as confirmed by doomfist players I've talked with)
4) Gravity when chaining abilities is _off_. You stay in low gravity for too long after using abilities
5) Arc slam too fast

This further proves my point that their doom feels smoother because their movement mechanic is really simple and doesn't perform any actual checks. For example, uppercut locking the player into animation is 100% intended because otherwise that mechanic is easily abusable to make him practically impossible to hit.
